testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/management/api/SocketsAndInterfacesTestCase.java

```diff
@@ -87,6 +89,30 @@
     @Before
     public void before() throws IOException {
         testNic = getNonDefaultNic();
+        // test the connector
+        testHost = NetworkUtils.canonize(testNic.getInetAddresses().nextElement().getHostName());
+    }
+    
+    @After
+    public void after() throws Exception{
+     // remove connector
+        ModelNode op = createOpNode("subsystem=undertow/server=default-server/http-listener=test", REMOVE);
+        ModelNode result = executeOperation(op);
+
+        try {
+            // check that the connector is down
+            Assert.assertFalse("Could not connect to created connector.",WebUtil.testHttpURL(new URL(
+                    "http", testHost, TEST_PORT, "/").toString()));
+        } finally {
+            // remove socket binding
+            op = createOpNode("socket-binding-group=standard-sockets/socket-binding=test-binding", REMOVE);
+            result = executeOperation(op);
+    
+            // remove interface
+            op = createOpNode("interface=test-interface", REMOVE);
```

Comment:
all this ops should be called with allow-resource-service-restart header so they don't need reload call at all and are removed at runtime.
see https://github.com/wildfly/wildfly/pull/8422/files#diff-e295371e5f22f6bc467299bf912c1faaR85
for example